engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -475,6 +475,58 @@ static void LogFrameBufferError(GLenum status)
 
     OpenGLContext* g_Context = 0x0;
 
+    static void UpdateGLHandle(uint32_t idx, GLuint handle)
+    {
+        g_Context->m_AllGLHandles[idx] = handle;
```

Comment:
Here and below - I use `g_Context` instead of passing `HContext` and cast it to `OpenGLContext` every time. Let me know if such approach is not acceptable.

Reply:
I reworked new API to get `HContext` as input parameter. But in the rest of code somewhere I need to use `g_Context` because no context pointer was passed as argument to parent function. If change old API (like `DeleteVertexBuffer`) we need to change lots of place where `gmGraphics::DeleteVertexBuffer` used and pass context as parameter. IMO it's out of scope of the current task. I can do it as a separate set of changes.